src/App.css

```diff
@@ -1,29 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+#root {
+  background-color: #1b1834;
+}
```

Comment:
- [ ] dear @Zainab-Alzubair, I see you take your way to style the layout of your book store and it looks great. But as part of the main requirement for this project we need to stick to the [Zeplin](https://zeplin.io/)[Zeplin](https://zeplin.io/) design. 

> Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) 

### Figma design 
![Screenshot 2022-12-24 204300](https://user-images.githubusercontent.com/69077061/209446541-2c1bd36c-f692-4aa9-8014-c17568f2abc0.png)

### Your design 
![Screenshot 2022-12-24 204738](https://user-images.githubusercontent.com/69077061/209446542-661bd245-d9bc-458d-bcb4-45013f1dc76d.png)

Reply:
Hi @ Tadesse-Alemaye , the design should include all the elements which I did but we are free to change the colors and the design

Reply:
Dear @Zainab-Alzubair, Actually as I reframe from the main requirements `Styling should be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/)`. Pleas try to make it look like the one on the given design.